File: Algorithms/C++/908-Smallest_Range_I.cpp
// Runtime: 12 ms, faster than 97.41% of C++ online submissions for Smallest Range I.
// Memory Usage: 9.7 MB, less than 75.00% of C++ online submissions for Smallest Range I.
class Solution {
public:
    int smallestRangeI(vector<int>& A, int K) {
        return max(*max_element(A.begin(), A.end()) - *min_element(A.begin(), A.end()) - 2 * K, 0);
    }
};

File: Algorithms/Python3.x/908-Smallest_Range_I.py
# Runtime: 140 ms, faster than 37.46% of Python3 online submissions for Smallest Range I.
# Memory Usage: 14.9 MB, less than 11.11% of Python3 online submissions for Smallest Range I.
class Solution:
    def smallestRangeI(self, A: List[int], K: int) -> int:
        return max(max(A) - min(A) - 2 * K, 0)
